order_guidelines.md

```diff
@@ -1,36 +1,36 @@
+Matroska requires only a few Top-Level Elements to be playable: `Info`, `Tracks`, and `Cluster`. These Elements have to be present in a Matroska file. The `Info` Element and `Tracks` Element <em>must</em> always be found before the `Cluster` Element(s) (either because they're located before the first `Cluster` or they can be found via `SeekHead` Elements; see below). All the other Elements can be omitted although the `Cues` Element (an index) can improve the playback experience greatly.
 
-After a Matroska file has been created it may still be edited. For example chapters, tags or cover art attachments can be added. To do that the Meta Seek needs to be updated and also some elements may be voided or extended. Therefore it's a good idea to use some padding after theses Elements. It is better to put that padding at the Level 1 so that they can easily be reshuffled without having to move them at the end.
+After a Matroska file has been created it may still be edited. For example chapters, tags or cover art attachments can be added. To do that the `SeekHead` Element(s) need to be updated and also some elements may be voided or extended. Therefore it's a good idea to use some padding after these Elements. It is better to put that padding at the Level 1 so that they can easily be reshuffled without having to move them at the end.
 
-## Meta Seek
+## SeekHead
 
-When 1 Meta Seek Head is present, it should be the first element in a Segment as it should reference the other level 1 elements in that segment (except maybe the long list of Clusters). There is also the possibility that in some cases there are two Meta Seek Head sections like for files produced with mkvpropedit. In that case the first one is also placed first with only the position of the level 1 elements except the Clusters. The second Meta Seek is placed at the end and contains a lengthy list of all Clusters (and not the other level 1 elements). Placing the first Meta Seek Head other than the first position of the Segment would make it a lot less useful So it <em>must</em> be the first element of a Cluster.
+When one `SeekHead` Element is present, it should be the first Element in a `Segment` as it should reference the other Top-Level Elements in that `Segment` (except maybe the long list of Clusters). There is also the possibility that in some cases there are two `SeekHead` Elements like for files produced with mkvpropedit. In that case the first `SeekHead` Element is also placed first with only the position of the Top-Level Elements except the `Cluster` Elements. The second `SeekHead` Element is placed at the end and contains a lengthy list of all `Cluster` Elements (and not the other Top-Level Elements). Placing the first `SeekHead` Element in a position other than the first position of the `Segment` would make it a lot less useful So it <em>must</em> be the first Element of a `Cluster`.
```

Comment:
fixed in https://github.com/Matroska-Org/matroska-specification/pull/11/commits/240384c8253f8e9b49963f001f1a4bffeeb91ef8.